### ISSUE TITLE
Accessibility Issue with Landing Page Carousel

### ISSUE THREAD

**Issue by daniel-cy-lu** [overture-stack/website#303]
The carousel is working. We would like to investigate accessibility of it.

**Comment by samrichca** [comment]
accessibility suggestions
- this article https://www.smashingmagazine.com/2023/02/guide-building-accessible-carousels/
  - TL;DR add aria labels and roles. don't have to worry about play/pause because it's not used here.
- for the slide selection images: these aren't keyboard-operable at the moment because they're images with click events. ideally we'd change these to button elements with images in them, but this may break a lot of CSS. an alternative would be [making the images focusable](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard) and [giving them a button role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role)
- for images in slides, change alt attributes to empty strings. current alt text is redundant with the nearby slide titles.

things that already look good in my testing:
- the order of elements in the page
- previous & next buttons
- inactive slides are hidden effectively
- react functionality of the carousel

goals:
- operate the carousel using the keyboard
- elements are labelled appropriately for screenreaders
